tests/publiccloud/az_accelerated_net.pm

```diff
@@ -0,0 +1,98 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Network performance for Azure Accelerated NICs
+#
+# Maintainer: Jose Lausuch <jalausuch@suse.de>
+
+use base "publiccloud::basetest";
+use strict;
+use testapi;
+use utils;
+use Data::Dumper;
+
+sub enable_accelerated_net {
+    my ($self, $instance) = @_;
+    my $name = $instance->{instance_id};
+
+    record_info('INFO', "Enabling accelerated network on instance '$name' ...");
+    assert_script_run("az vm deallocate --resource-group $name --name $name",                                        timeout => 60 * 10);
```

Comment:
Only to understand, would a `az vm stop` work as well?

Reply:
I Was following the documentation in https://docs.microsoft.com/en-us/azure/virtual-network/create-vm-accelerated-networking-cli#individual-vms--vms-in-an-availability-set

It actually says stop/deallocate. I will try to change it to "stop" command. Not sure what the difference is.

Reply:
ok, thx for investigation.